lib/active_record/connection_adapters/oracle_enhanced_adapter.rb

```diff
@@ -103,7 +103,7 @@ module ConnectionAdapters #:nodoc:
     # Ruby 1.8/1.9 ruby-oci8 gem (which provides interface to Oracle OCI client)
     # or with JRuby and Oracle JDBC driver.
     #
-    # It should work with Oracle 9i, 10g and 11g databases.
+    # It should work with Oracle 9i, 10g, 11g and 12c databases.
     # Limited set of functionality should work on Oracle 8i as well but several features
     # rely on newer functionality in Oracle database.
```

Comment:
Recent changes do not consider older version of Oracle 8i or 9i, such as virtual columns and TIMESTAMP data type . Then let's remove documentation mentioning 9i or older versions of Oracle.